go.mod

```diff
@@ -3,6 +3,8 @@ module github.com/gptscript-ai/gptscript
 // This can't be upgraded until the issue with sys.daemon on Windows is resolved
 go 1.22.4
 
+replace github.com/gptscript-ai/tui => ../tui
```

Comment:
Just a note to drop this before merging

Reply:
This was a mistake and has been removed.